modules/serverpod_auth/serverpod_auth_server/generated/migration/migrations/migration_registry.txt

```diff
@@ -4,4 +4,4 @@
 ### manually. If a collision is detected in this file when doing a code merge, resolve
 ### the conflict by removing and recreating the conflicting migration.
 
-20231205080924753
+20240104145916156
```

Comment:
Suggestion: Add as a new migration if needed.

This needs to be added as a new migration since developers could have created a dependency towards any module in the RC release. We should probably support those created projects if there is nothing critical hindering us from doing it. But we might not need to update it if we keep the old syntax and remove "CREATE IF NOT EXISTS...".

Reply:
Had a quick check since I was thinking the same on the format as commented above, but I don't think we strictly have to have no breaking changes in an rc by semver standard at least

> pre-release version indicates that the version is unstable and might not satisfy the intended compatibility requirements as denoted by its associated normal version.

Now if we remove the `IF NOT EXISTS` there should be no changes in the migration files anyway.

Reply:
Aah awesome, great to have that flexibility! 👍